src/main/java/org/broadinstitute/hellbender/tools/ApplyBQSR.java

```diff
@@ -57,6 +58,9 @@ public void onTraversalStart() {
         outputWriter = new SAMFileGATKReadWriter(new SAMFileWriterFactory().makeWriter(outputHeader, true, OUTPUT, referenceArguments.getReferenceFile()));
         transform = new BQSRReadTransformer(outputHeader, BQSR_RECAL_FILE, bqsrArgs);
         Utils.warnOnNonIlluminaReadGroups(getHeaderForReads(), logger);
+        if (! bqsrArgs.disableIndelQuals) {
+            throw new UserException.BadArgumentValue("disable_indel_quals", "false", "GATK4 does not support BQSR on indels");
```

Comment:
it's kept for backwards compatibility. The unfortunate thing is that the default was false (as it should have been) so removing it would change the default which is kind of confusing. So i'd rather keep it and force people to pass it on. @vdauwera What do you think?